src/Chainweb/Pact/Backend/InMemoryCheckpointer.hs

```diff
@@ -67,6 +67,7 @@ initInMemoryCheckpointEnv loggers logger = do
                     , _cpGetBlockParent = doGetBlockParent inmem
                     , _cpRegisterProcessedTx = doRegisterSuccessful inmem
                     , _cpLookupProcessedTx = doLookupSuccessful inmem
+                    , _cpGetBlockHistory = undefined -- TODO
```

Comment:
Is this for future work?

Reply:
changed to "unimplemented", this checkpointer is not used in production